.github/workflows/jstest.yml

```diff
@@ -1,15 +1,45 @@
 name: TestJS
 
 on:
-  # Run on pushes to select branches and on all pull requests.
+  # Run on relevant pushes to select branches and on all relevant pull requests.
   push:
     branches:
       - main
       - trunk
       - 'release/**'
       - 'hotfix/[0-9]+.[0-9]+*'
       - 'feature/**'
+    paths:
+      - '**.js' # Includes Gruntfile.js.
+      - '.browserslistrc'
+      - '.eslintignore'
+      - '.eslintrc'
+      - '.nvmrc'
+      - '.yarnrc'
+      - 'lerna.json'
+      - 'package.json'
+      - 'yarn.lock'
+      - '.github/workflows/js.yml'
+      - '.yarn/**'
+      - 'apps/**'
+      - 'config/grunt/**'
+      - 'packages/**'
   pull_request:
+    paths:
+      - '**.js' # Includes Gruntfile.js.
+      - '.browserslistrc'
+      - '.eslintignore'
+      - '.eslintrc'
+      - '.nvmrc'
+      - '.yarnrc'
+      - 'lerna.json'
+      - 'package.json'
+      - 'yarn.lock'
+      - '.github/workflows/js.yml'
```

Comment:
shoulded this be - '.github/workflows/jstest.yml' ?
and maybe also:
- '.github/workflows/jslint.yml' ?